dev/app/src/product-detail.html

```diff
@@ -24,7 +24,14 @@
 </head>
 
 <body>
-
+<script>
+  let productId = new URLSearchParams(window.location.search).get('email');
```

Comment:
this part(27~33) is usless(it is just test) don't care about it(I will delete this soon)